macOS/DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/NetworkProtectionNavBarPopoverManager.swift

```diff
@@ -116,22 +116,27 @@ final class NetworkProtectionNavBarPopoverManager: NetPPopoverManager {
                 }))
         }
 
+        if proxySettings.proxyAvailable {
```

Comment:
This is just to hide exclusion menu options if the proxy is not available.  It's the menu options shown here, which should not be visible in App Store.

<img width="422" alt="Screenshot 2025-03-17 at 12 21 23 PM" src="https://github.com/user-attachments/assets/6ba472cd-f875-44f0-b02a-9aec704f57b5" />